addons/web/static/src/env.js

```diff
@@ -1,8 +1,9 @@
 /** @odoo-module **/
 
```

Comment:
feels like this file is not really about the env, but more about starting an app. maybe rename it to start_utils.js? and possibly move the content of `start.js` to `main.js`

Reply:
This is "stable" so I'd rather not move things around, but open to the idea in master.

Reply:
oh right. usually, i wouldn't do that kind of work in stable, but maybe it makes sense to do it for documentation/onboarding/training material.

Reply:
Yes, this is also why I only updated the webclient app creation, so that the code isn't dead but we keep the footprint of changes to existing code minimal with no risk of breaking customizations.